letsdrill.rb

```diff
@@ -1,24 +1,28 @@
-def get_letter_grade(integer)
- 
-  #Put your code here!
-
+def get_letter_grade(score)
+  case score
+  when 90..100
+    return "A"
```

Comment:
Ah, right. I always get Ruby's implementation of the case statement confused with the way switches work in other languages ([fall through behavior](http://www.codecademy.com/glossary/ruby/switch-statement)).